src/main/java/uk/gov/digital/ho/proving/income/domain/Individual.java

```diff
@@ -1,44 +1,28 @@
 package uk.gov.digital.ho.proving.income.domain;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.Accessors;
+
+@AllArgsConstructor
+@Getter
+@Accessors(fluent = true)
+@ToString
 public class Individual {
-    private String title;
+    @JsonProperty
```

Comment:
I thought we were going to add the name of the json field to this annotation. If we don't the annotation is (sort of) superfluous.

Reply:
Done, plus added serialization tests.